nameko_sqlalchemy.py

```diff
@@ -22,11 +22,14 @@ def setup(self):
             'service_name': service_name,
             'declarative_base_name': decl_base_name,
         })
+        self.engine = create_engine(self.db_uri)
+
+    def stop(self):
+        self.engine.dispose()
```

Comment:
That "very simple test" is the hard bit ;) The effect of not disposing of the engine is that you get a connection left lying around until the `DependencyProvider` is garbage collected. sqlalchemy doesn't make it easy to inspect the number of connections though.

I will change the code so its more locally testable.